src/portal/lib/src/tag/tag.component.spec.ts

```diff
@@ -122,16 +133,17 @@ describe("TagComponent (inline template)", () => {
       providers: [
         ErrorHandler,
         ChannelService,
+        HttpClient,
```

Comment:
You can just import HttpClientModule from angular test. import { HttpClientTestingModule } from '@angular/common/http/testing';
imports: [
            HttpClientTestingModule
            ],

Reply:
fixed